debugd/internal/debugd/deploy/seriallogin.go

```diff
@@ -1,41 +0,0 @@
-/*
-Copyright (c) Edgeless Systems GmbH
-
-SPDX-License-Identifier: AGPL-3.0-only
-*/
-
-package deploy
-
-import (
-	"context"
-	"fmt"
-	"os"
-	"path"
-
-	"github.com/edgelesssys/constellation/v2/debugd/internal/debugd"
-	"github.com/spf13/afero"
-)
-
-// EnableAutoLogin installs a systemd unit override that allows passwordless root login
-// on the serial console.
-func EnableAutoLogin(ctx context.Context, fs afero.Fs, serviceManager serviceManager) error {
-	if err := fs.MkdirAll(path.Dir(debugd.GettyAutologinOverrideFilename), os.ModePerm); err != nil {
-		return fmt.Errorf("creating getty autologin override directory: %w", err)
-	}
-	if err := afero.WriteFile(fs, debugd.GettyAutologinOverrideFilename,
-		[]byte(debugd.GettyAutologinOverrideUnitContents), os.ModePerm); err != nil {
```

Comment:
You can likely also remove the two Getty constants in `constants.go` --> https://github.com/edgelesssys/constellation/blob/67f8336b9d2c3e935345559503053339fc1c25ad/debugd/internal/debugd/constants.go#L34